### PR TITLE
 ✨ New `payments` service (⚠️ devops)

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Basic skeleton for a new `services/payments` fastapi-based service including:
- minimal core, api, testing
- added in osparc stack
- ci jobs

Next PR will implement the endpoints in the `web-server` for the payment transaction.

## Related issue/s

- Implementing 4657

## How to test
```cmd
cd services/payment
make install-dev
make tests-dev-unit
```

## DevOps 

- New service variables
```.env
PAYMENTS_HOST=invitations
PAYMENTS_LOGLEVEL=INFO
PAYMENTS_GATEWAY_URL=http://fake-payment-gateway.com # For the moment keep it fake
PAYMENTS_PASSWORD=adminadmin
PAYMENTS_PORT=8000
PAYMENTS_SECRET_KEY='REPLACE_ME_with_result__Fernet_generate_key='
PAYMENTS_USERNAME=admin
```

### CODE REVIEW

File: .github/workflows/ci-testing-deploy.yml [hunk 1]
```diff
@@ -917,6 +924,45 @@ jobs:
         with:
           flags: unittests #optional
 
+  unit-test-payments:
```
Comment (sanderegg) [inline .github/workflows/ci-testing-deploy.yml:927]: I think you missed to add it in the unit-tests job

File: packages/models-library/src/models_library/api_schemas_payments/meta.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    name: str
+    version: VersionStr
+    docs_url: HttpUrl
+
+    class Config:
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "example": {
+                "name": "simcore_service_payments",
+                "version": "2.4.45",
+                "docs_url": "https://foo.io/doc",
```
Comment (sanderegg) [inline packages/models-library/src/models_library/api_schemas_payments/meta.py:18]: foo.io???

File: services/payments/requirements/_test.in [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+--constraint _base.txt
+
+
+coverage
+faker
+httpx
+pytest
+pytest-asyncio
+pytest-cov
+pytest-runner
```
Comment (sanderegg) [inline services/payments/requirements/_test.in:20]: if you like the nice dict diff.. I think you need to add pytest-icdiff

File: services/payments/src/simcore_service_payments/_meta.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+API_VTAG: Final[str] = info.api_prefix_path_tag
+SUMMARY: Final[str] = info.get_summary()
+
+
+# NOTE: https://patorjk.com/software/taag/#p=testall&f=Standard&t=Payments
+APP_STARTED_BANNER_MSG = r"""
+ ____   __   _  _  _  _  ____  __ _  ____  ___
+(  _ \ / _\ ( \/ )( \/ )(  __)(  ( \(_  _)/ ___)
+ ) __//    \ )  / / \/ \ ) _) /    /  )(  \___ \
+(__)  \_/\_/(__/  \_)(_/(____)\_)__) (__) (____/  {}
```
Comment (sanderegg) [inline services/payments/src/simcore_service_payments/_meta.py:25]: ahahah

File: services/docker-compose.local.yml [hunk 5]
```diff
@@ -62,6 +62,10 @@ services:
     ports:
       - "8008:8000"
 
+  payments:
+    ports:
+      - "8011:8000"
```
Comment (sanderegg) [inline services/docker-compose.local.yml:67]: debug port?

**Review by sanderegg — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]
thanks!

File: services/docker-compose.local.yml (hunk 5 above)
Comment (pcrespov) [reply services/docker-compose.local.yml:67]: yes

File: services/docker-compose.yml [hunk 6]
```diff
@@ -195,6 +195,20 @@ services:
       - INVITATIONS_PASSWORD=${INVITATIONS_PASSWORD}
       - LOG_FORMAT_LOCAL_DEV_ENABLED=${LOG_FORMAT_LOCAL_DEV_ENABLED}
 
+  payments:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/payments:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    networks:
+      - default
```
Comment (mrnicegyu11) [inline services/docker-compose.yml:203]: who does this guy talk to actually? (microservices/ outside?)

**Review by mrnicegyu11 — APPROVED** [review]
didnt read all, the ops parts,.

File: services/docker-compose.yml (hunk 6 above)
Comment (pcrespov) [reply services/docker-compose.yml:203]: webserver, postgres and RUT

**Review by GitHK — APPROVED** [review]
looking good, this is just a base I see.

**Pull request merged**